Tests failed after updating should.js to 9.*

```javascript
1) toshihiko #execute select rows:
    Uncaught AssertionError: expected Array [ TextRow { id: 1, ls: 1 } ] to equal Array [ Object { id: 1, ls: 1 } ] (at '0', A has TextRow { id: 1, ls: 1 } and B has Object { id: 1, ls: 1 } because A and B have different prototypes)
```

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "cliparoo": "^1.1.1",
     "conventional-changelog": "^0.5.3",
     "coveralls": "^2.11.6",
-    "cz-conventional-changelog": "^1.1.5",
+    "cz-conventional-changelog": "^1.1.6",
     "ghooks": "^1.2.2",
     "istanbul": "^1.0.0-alpha.2",
     "jsxhint": "^0.15.1",

--- a/test/toshihiko.js
+++ b/test/toshihiko.js
@@ -5,6 +5,7 @@
  * reserved.
  */
 var should = require("should");
+should.config.checkProtoEql = false;
 
 var Toshihiko = require("../lib/toshihiko");
 var Type = require("../lib/field_type");
@@ -42,7 +43,7 @@ describe("toshihiko", function() {
                 should(err).equal(null);
                 data.should.be.an.instanceOf(Array);
                 data.should.have.length(1);
-                data.should.eql([{
+                data.should.be.eql([{
                     id: 1,
                     ls: 1
                 }]);
@@ -90,4 +91,3 @@ describe("toshihiko", function() {
 
     });
 });
-

